src/components/Envelope.vue

```diff
@@ -59,26 +59,47 @@
 		</template>
 		<template #actions>
 			<EnvelopePrimaryActions v-if="!moreActionsOpen">
-				<ActionButton icon="icon-starred"
+				<ActionButton
 					class="action--primary"
 					:close-after-click="true"
 					@click.prevent="onToggleFlagged">
+					<template #icon>
+						<Star v-if="iconFavorite"
+							:title="t('mail', 'Favorite')"
+							:size="20" />
+						<StarOutline v-else
+							:title="t('mail', 'Unfavorite')"
+							:size="20" />
+					</template>
 					{{
 						data.flags.flagged ? t('mail', 'Unfavorite') : t('mail', 'Favorite')
 					}}
 				</ActionButton>
-				<ActionButton icon="icon-mail"
+				<ActionButton
 					class="action--primary"
 					:close-after-click="true"
 					@click.prevent="onToggleSeen">
+					<template #icon>
+						<Email v-if="iconImportant"
+							:title="t('mail', 'Unread')"
+							:size="20" />
+						<EmailOutline v-else
+							:title="t('mail', 'Read')"
+							:size="20" />
+					</template>
 					{{
 						data.flags.seen ? t('mail', 'Unread') : t('mail', 'Read')
 					}}
 				</ActionButton>
-				<ActionButton icon="icon-important"
+				<ActionButton
 					class="action--primary"
 					:close-after-click="true"
 					@click.prevent="onToggleImportant">
+					<template #icon>
+						<ImportantIcon
+							:title="isImportant ? t('mail', 'Unimportant') : t('mail', 'Important')"
```

Comment:
I think we can omit the title here. The action button already has a text which should be informative enough for screen readers.

This also applies to all other icon texts.